### DISCUSSION TITLE
[Esame 27 Gennaio 2023] Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#344] [Esami - Bandiera]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/7f0b4151-c85b-4368-96d3-53b4b9950666)

**Comment by Elia-Belli** [comment]
**(a)** Osserviamo che $(12,33)=3|21$ , $(35,25)=5|20$ e $(21,39)=3|33$ allora il sistema è risolvibile
Riscriviamolo in modo equivalente come (dividendo per gli MCD calcolati):

$$\begin{cases}4x\equiv7(11)\\
5x\equiv 4(7)\\
7x\equiv 11(13)\end{cases}$$

Siccome i moduli solo numeri primi, e quindi coprimi a coppie, è possibile passare al sistema cinese (moltiplico ambo i lati di ogni eq. per l'inverso di $a$) e semplificare:

$$\begin{cases}x\equiv21(11)\\
x\equiv 12(7)\\
x\equiv 22(13)\end{cases}\to\begin{cases}x\equiv10(11)\\
x\equiv 5(7)\\
x\equiv 9(13)\end{cases}$$

Ora per il Teorema del Resto cinese la soluzione è data da $\tilde{x}=R_1\tilde{x}_1+R_2\tilde{x}_2+R_3\tilde{x}_3\ (mod\ R)$
con $R=1001$ , $R_1=91$ , $R_2=143$ ed $R_3=77$ e le incognite date risolvendo le seguenti equazioni congruenziali:

$$91\tilde{x}_1\equiv 10(11)\to 3\tilde{x}_1\equiv 10(11)\to \tilde{x}_1\equiv40(11)\to \tilde{x}_1\equiv 7(11)$$

$$143\tilde{x}_2\equiv 5(7)\to 3\tilde{x}_2\equiv 5(7)\to \tilde{x}_2\equiv25(7)\to \tilde{x}_2\equiv 4(7)$$


$$77\tilde{x}_3\equiv 9(13)\to 12\tilde{x}_3\equiv 9(13)\to \tilde{x}_3\equiv-9(13)\to \tilde{x}_3\equiv 4(13)$$

Quindi $\tilde{x}=91\cdot 7+143\cdot 4+77\cdot 4=1517\ (1001)=516\ (1001)$

**(b)** Tale sistema non è in forma cinese (ci sono dei moduli non coprimi), ciò non vuol dire che non esista soluzione quindi procediamo per sostituzione partendo da $x=5+18t_1$ ottenuto dalla prima equazione e inserendolo nella seconda:
$$5+18t_1\equiv 17(21)\to t_1\equiv 3(7)\to t_1=3+7t_2$$
"Aggiorniamo" la soluzione $x=5+18(3+7t_2)=59+126t_2$ e la inseriamo nella terza:
$$59+126t_2=23(24)\to 6t_2\equiv 12(24)\to t_2\equiv 2(4)\to t_2=2+4t_3$$
Infine $x=59+126(2+4t_3)=311+504t_3\to x\equiv 311(504)$

**(c)** Il terzo sistema non è in forma cinese, allora procediamo per sostituzione:
$$x=7+18t_1$$

$$7+18t_1\equiv 13(21)\to 18t_1\equiv 6(21)\to 9t_1\equiv 3(7)\to t_1\equiv 5(7)$$

$$x=7+18(5+7t_2)=97+126t_2$$

$$97+126t_2\equiv 11(24)\to 126t_2\equiv -86(24)\to 6t_2\equiv 10(24)\to (24,6)=6\ \nmid 10\Rightarrow no\ soluzioni$$

**Reply by IonutCicio** [reply]
> [!CAUTION]
> Attento ai punti $(b)$ e $(c)$

È vero che in questo caso $(b)$ e $(c)$ non hanno soluzione[^1] , ma non è perché $18$ e $21$ non sono coprimi. Il teorema cinese del resto dice solamente che se il sistema ha quella forma, e tutti gli $n_i$ sono coprimi fra loro, allora la soluzione c'è sicuramente _(ed è unica, in una forma ben precisa)_.

Se gli $n_i$ non sono coprimi fra loro, non ti garantisce che la soluzione non esiste! Semplicemente non sai se esiste o meno, e devi procedere per sostituzione.

Infatti, se consideri il punto $(c)$ dell'esercizio #354 si ha che $(36, 40) = 4 \neq 1$, ma il sistema ha comunque soluzione $491$

[^1]: Avevo fatto male i conti: (b) ha soluzione

**Reply by cornflexxx** [reply]
Quindi, se non ho capito male, nella $(b)$ una soluzione c'è perché si può scomporre così:

$$\left\\{ \begin{array}{cl}
x\equiv 5(18) \\
x\equiv 17(21)\\
x\equiv 23(24)
\end{array} \right.\longrightarrow \left\\{ \begin{array}{cl}
x\equiv 5(3) \\
x\equiv 5(2)\\
x\equiv 17(7)\\
x\equiv 17(3)\\
x\equiv 23(2)\\
x\equiv 23(3)
\end{array} \right.\longrightarrow \left\\{ \begin{array}{cl}
x\equiv 2(3) \\
x\equiv 1(2)\\
x\equiv 3(7)\\
x\equiv 2(3)\\
x\equiv 1(2)\\
x\equiv 2(3)
\end{array} \right.\rightarrow \left\\{ \begin{array}{cl}
x\equiv 2(3) \\
x\equiv 1(2)\\
x\equiv 3(7)\\
\end{array} \right. $$

Quindi esiste un'unica soluzione  a questo sistema $mod(3\cdot 2\cdot 7)\rightarrow [17]_{42}$

--------------------
Quando invece scomponiamo la $(c)$, otteniamo  un sistema di equazioni con moduli coprimi ma resti discordanti, quindi non esiste soluzione.


$$\left\\{ \begin{array}{cl}
x\equiv 7(18) \\
x\equiv 13(21)\\
x\equiv 11(24)
\end{array} \right.\longrightarrow \left\\{ \begin{array}{cl}
x\equiv 7(3) \\
x\equiv 7(2)\\
x\equiv 13(7)\\
x\equiv 13(3)\\
x\equiv 11(2)\\
x\equiv 11(3)
\end{array} \right.\longrightarrow \left\\{ \begin{array}{cl}
x\equiv 1(3) \\
x\equiv 1(2)\\
x\equiv 6(7)\\
x\equiv 1(3)\\
x\equiv 1(2)\\
x\equiv 2(3)
\end{array} \right.\rightarrow \text{non ammette soluzione}$$

**Reply by Frisbon** [reply]
@Elia-Belli Nel punto b, perché hai selezionato come modulo finale 1512?
Se non sbaglio, non dovrebbe essere l'mcm(18,21,24)?

**Reply by Elia-Belli** [reply]
@Frisbon errore di calcolo, grazie

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/738c497c-9055-4c72-b091-41f43100da97)
